rust/ios/src/lib.rs

```diff
@@ -1,378 +0,0 @@
-use std::ffi::{CStr, CString};
-use std::os::raw::c_char;
-
-#[no_mangle]
-pub extern "C" fn rust_free(s: *mut c_char) {
```

Comment:
@AloeareV do you think this method make any sense to use in the unique rust layer now? It seems like for the removed rust ios layer clean the memory...